packages/flame/lib/src/post_process/post_process.dart

```diff
@@ -0,0 +1,138 @@
+import 'dart:async';
+import 'dart:ui' as ui show Image;
+import 'dart:ui' hide Image;
+
+import 'package:flame/components.dart';
+import 'package:flame/post_process.dart';
+import 'package:meta/meta.dart';
+
+abstract class PostProcess {
+  PostProcess({double? pixelRatio})
```

Comment:
Could this be a component? :thinking:
With a `ParentIsA` mixin on it to restrict what it can be added to?

Reply:
Ah, or maybe not, since it is added to the decorator in `PostProcessComponent` and not rendered directly.